bundles/org.dataflowanalysis.analysis.converter/src/org/dataflowanalysis/analysis/converter/microsecend/TaggedValuesDeserializer.java

```diff
@@ -0,0 +1,47 @@
+package org.dataflowanalysis.analysis.converter.microsecend;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonToken;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class TaggedValuesDeserializer extends JsonDeserializer<Map<String, List<String>>> {
+
+    @Override
+    public Map<String, List<String>> deserialize(JsonParser parser, DeserializationContext ctxt) throws IOException {
```

Comment:
The method does not throw an `IOException`, and thus the throws clause can be omitted

Reply:
`parser.nextToken()`throws an io exception

Reply:
Correct. It seems I was missing the Jackson Plugin and didn't notice the errors on my side :D